docs/modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -146,7 +150,7 @@ To enable the writing of core dump files, you must define the `OWNCLOUD_CORE_DUM
 
 For example:
 
-`\` OWNCLOUD_CORE_DUMP=1 owncloud``
```

Comment:
The call actually should be `OWNCLOUD_CORE_DUMP=1 owncloud`

Reply:
Fixed